ibis/expr/api.py

```diff
@@ -1263,29 +1264,18 @@ def _integer_to_interval(arg, unit='s'):
     return op.to_expr()
 
 
-abs = _unary_op('abs', _ops.Abs)
-ceil = _unary_op('ceil', _ops.Ceil)
-exp = _unary_op('exp', _ops.Exp)
-floor = _unary_op('floor', _ops.Floor)
-log2 = _unary_op('log2', _ops.Log2)
-log10 = _unary_op('log10', _ops.Log10)
-ln = _unary_op('ln', _ops.Ln)
-sign = _unary_op('sign', _ops.Sign)
-sqrt = _unary_op('sqrt', _ops.Sqrt)
```

Comment:
Don't remove these, since they are a breaking API change. If you want to deprecate them, open an issue about it and we can discuss there.